Agreements/Internal/13_D.txt

```diff
@@ -1,6 +1,11 @@
 COPX:*:*:LAM:EGBB:*:London S13:London AC Dover:*:27000:LAM35
 COPX:*:*:LAM:EGBE:*:London S13:London AC Dover:*:27000:LAM35
 COPX:*:*:LAM:EGNX:*:London S13:London AC Dover:*:27000:LAM35
+
 COPX:*:*:LAM:EGGD:*:London S13:London AC Dover:*:30000:LAM35
 COPX:*:*:LAM:EGFF:*:London S13:London AC Dover:*:30000:LAM35
 COPX:*:*:LAM:EGSY:*:London S13:London AC Dover:*:30000:LAM35
+
+COPX:*:*:TRIPO:EGGD:*:London S13:London AC Dover:*:30000:TRIPO
+COPX:*:*:TRIPO:EGFF:*:London S13:London AC Dover:*:30000:TRIPO
+COPX:*:*:TRIPO:EGSY:*:London S13:London AC Dover:*:30000:TRIPO
```

Comment:
everyone goes via LAM but not everyone via TRIPO, so the TRIPO agreement should be above the LAM35 agreement